mainwindow.cpp

```diff
@@ -308,7 +308,7 @@ void MainWindow::reprocess_image(int scale, int quality) {
             return;
         }
     }
-
+    m_out_data.clear();
```

Comment:
before, when quality slider value is reduiced, image size couldn'd change. now it is changeable.